dev/service-registry/client-javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/registry/RegisteredService.java

```diff
@@ -26,9 +22,7 @@
  * {@code RegisteredService.of()}.
  */
 @SuppressWarnings("all")
-@ParametersAreNonnullByDefault
 @Generated({"Immutables.generator", "AbstractRegisteredService"})
```

Comment:
To be honest, I think we should replace these classes with hand written ones.

As I understand it, Immutables looks at your classpath, works out what annotations are present, and then uses them if they are present.  These annotations I think come from the findbugs javax annotations jar, which is no longer on the classpath after the Play 2.6 upgrade, and I didn't think it worth adding them just so these classes could still use them.  So I just deleted the annotations.